dtfs-central-api/src/v1/controllers/utilisation-report-service/post-upload-utilisation-report.controller/index.ts

```diff
@@ -8,11 +9,10 @@ import {
 } from '../../../validation/utilisation-report-service/utilisation-report-validator';
 import { UtilisationReportStateMachine } from '../../../../services/state-machines/utilisation-report/utilisation-report.state-machine';
 import { CustomExpressRequest } from '../../../../types/custom-express-request';
-import { AzureFileInfo } from '../../../../types/azure-file-info';
 import { UtilisationReportRawCsvData } from '../../../../types/utilisation-reports';
 import { ApiError } from '../../../../errors';
 
-type PostUploadUtilisationReportRequest2 = CustomExpressRequest<{
+export type PostUploadUtilisationReportRequest2 = CustomExpressRequest<{
```

Comment:
What's the need for this type over the other one? Is it purely for the payloadValidator before we cast to the other type after? Reckon we can improve the names if so

Reply:
Yeh this was something Mike left behind - I believe it just takes everything in `PostUploadUtilisationReportRequest` and sets everything to `unknown`. I'll have a think about a better way of doing this